contrib/guix/libexec/build.sh

```diff
@@ -368,6 +368,14 @@ mkdir -p "$DISTSRC"
                 ;;
         esac
 
+        # If a bitcoin.conf file has been generated, copy it over
+        if [ -f "${DISTSRC}/share/examples/bitcoin.conf" ]
```

Comment:
Now that the placeholder is always there, this if/else is no longer necessary and can be replaced with just a copy.

Reply:
fixed!